apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableByteStringTest.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Protobuf;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Google.Cloud.Bigtable.V2.Tests
+{
+    public class BigtableByteStringTest
+    {
+        [Fact]
+        public void EmptyString()
+        {
+            BigtableByteString control = default;
+            var equal = new[]
+            {
+                new BigtableByteString((string)null),
+                new BigtableByteString((byte[])null),
+                new BigtableByteString((ByteString)null),
+                new BigtableByteString(""),
+                new BigtableByteString(new byte[0]),
+                new BigtableByteString(ByteString.Empty)
+            };
+            var unequal = new[]
+            {
+                new BigtableByteString(0),
+                new BigtableByteString(0L)
```

Comment:
Maybe add non-empty strings and ByteStrings here?

Reply:
Done